src/commands/Tools/Dictionary/define.ts

```diff
@@ -0,0 +1,87 @@
+import { KlasaMessage } from 'klasa';
+import { TOKENS } from '@root/config';
+import { DbSet } from '@lib/structures/DbSet';
+import { RichDisplayCommand, RichDisplayCommandOptions } from '@lib/structures/RichDisplayCommand';
+import { UserRichDisplay } from '@lib/structures/UserRichDisplay';
+import { cutText, fetch, FetchResultTypes } from '@utils/util';
+import { toTitleCase } from '@klasa/utils';
+import { MessageEmbed } from 'discord.js';
+import { ApplyOptions } from '@skyra/decorators';
+import { BrandingColors, Mime } from '@utils/constants';
+
+@ApplyOptions<RichDisplayCommandOptions>({
+	aliases: ['definition', 'defination', 'dictionary'],
+	bucket: 2,
+	cooldown: 20,
+	description: language => language.get('COMMAND_DEFINE_DESCRIPTION'),
+	extendedHelp: language => language.get('COMMAND_DEFINE_EXTENDED'),
+	usage: '<input:string>'
+})
+export default class extends RichDisplayCommand {
+
+	public async run(message: KlasaMessage, [input]: [string]) {
+		const response = await message.sendEmbed(new MessageEmbed()
+			.setDescription(message.language.tget('SYSTEM_LOADING'))
+			.setColor(BrandingColors.Secondary));
+
+		const result = await this.fetchApi(message, input);
```

Comment:
What does the API return when there is no definition? 400/500 errors won't be caught by the catch. If so then handle it here. See for example https://github.com/skyra-project/skyra/blob/9a6b1d0fe27c82f9b7ed461bed724d750e56cd12/src/commands/Anime/anime.ts#L30-L35

Reply:
It actually does catch 400/500 errors because I'm awaiting the response within the method as it's assigned to the variable.

Thus it'll catch the error, also I've tested it.